examples/yoti_example_flask/templates/profile.html

```diff
@@ -1,382 +1,124 @@
+{% macro parse_documents(prop) %}
```

Comment:
Since we also have a `DocumentDetails` attribute (yet to be added to Python), it might be slightly clearer if this was called either `parse_images` or `parse_document_images` 👍 
```suggestion
{% macro parse_document_images(prop) %}
```

Reply:
Probably best to not commit it here in the GUI, as it'll obviously need to be updated where it's called as well